docs/README

```diff
@@ -0,0 +1,27 @@
+# Welcome to MkDocs
```

Comment:
Can we add a .md extension to this file?

Also, explain here that the user must install mkdocs-material, otherwise this all doesn't work.

Reply:
In fact, we should probably just replace it without our own test if it's gonna show up on opening the `docs/` folder on github.

Reply:
> Can we add a .md extension to this file?

Yes. And yes I will add the step for installing mkdocs-material. Thanks.  :)

@rchl  I don't have enough context to get what you wanted to say. Can you clarify `it` and `test`? 

> In fact, we should probably just replace **it** without our own **test** if it's gonna show up on opening the docs/ folder on github.

Reply:
Meant to say "text", not "test" but never mind me. I no longer believe what I've said mysefl.